addons/product/views/product_packaging_views.xml

```diff
@@ -1,96 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<odoo>
-
-    <record id="product_packaging_tree_view" model="ir.ui.view">
-        <field name="name">product.packaging.list.view</field>
-        <field name="model">product.packaging</field>
-        <field name="arch" type="xml">
-            <list string="Product Packagings" name="packaging">
-                <field name="sequence" widget="handle"/>
-                <field name="product_id"/>
-                <field name="name" string="Packaging"/>
-                <field name="qty"/>
-                <field name="product_uom_id" options="{'no_open': True, 'no_create': True}" groups="uom.group_uom"/>
-                <field name="barcode" optional="hide"/>
```

Comment:
We're probably going to want to show the new "packaging" barcodes in a list view, otherwise it's going to be a nightmare to keep track of them since you're blocked from duplicating a barcode, but you don't know which "packaging" is using it